src/test/fakes.rs

```diff
@@ -364,19 +364,20 @@ struct FakeGithubStats {
 }
 
 impl FakeGithubStats {
-    fn create(&self, conn: &mut Client) -> Result<String, Error> {
+    fn create(&self, conn: &mut Client) -> Result<i32, Error> {
         let existing_count: i64 = conn
-            .query_one("SELECT COUNT(*) FROM github_repos;", &[])?
+            .query_one("SELECT COUNT(*) FROM repositories;", &[])?
             .get(0);
-        let id = base64::encode(format!("FAKE ID {}", existing_count));
+        let host_id = base64::encode(format!("FAKE ID {}", existing_count));
 
-        conn.execute(
-            "INSERT INTO github_repos (id, name, description, last_commit, stars, forks, issues, updated_at)
-             VALUES ($1, $2, 'Fake description!', NOW(), $3, $4, $5, NOW());",
-            &[&id, &self.repo, &self.stars, &self.forks, &self.issues],
+        let data = conn.query(
```

Comment:
```suggestion
        let data = conn.query_one(
```

Reply:
Oh nice. I updated the other `INSERT INTO` I modified in this PR too.